app/main.py

```diff
@@ -1 +1,10 @@
-# write your code here
+def copy_file(command: str) -> None:
+    if len(command.split()) != 3:
+        return print("Command length is not equal to 3")
+
+    action, original_file, new_file = command.split()
+    if action != "cp" or original_file == new_file:
```

Comment:
I disagree about second part `original_file == new_file ` as your code should do nothing in this particular situation but you give a message about incorrect command. It could lead to misunderstanding